demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -187,7 +188,7 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver,
           }
           (document.getElementById(
             'meeting-id'
-          ) as HTMLSpanElement).innerHTML = `${this.meeting} (${this.region})`;
+          ) as HTMLSpanElement).innerHTML = `${meetingId} ${this.meeting} (${this.region})`;
```

Comment:
I tested this out and it seems the id can get really long and will be shown under the button so it is very hard to read.
Maybe we could move this down to the bottom of the page between the up and down link?

Reply:
It is visible when the window if maximized. Sure, can be moved down